### ISSUE TITLE
[Samba NAS] Can't mount drive even with protection mode off

### ISSUE THREAD

**Issue by marciogranzotto** [dianlight/hassio-addons#3]
I'm running HassOS 4.8, Supervisor v225, on an Intel NUC.

I'm running the add-on with protection mode turned off, but it still fails to mount my exfat USB drive.
```shell
[19:11:17] INFO: Hostname: homeassistant
[19:11:17] INFO: More Disks mounting.. wd-drive
[19:11:17] INFO: Mount wd-drive
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
fuse: device not found, try 'modprobe fuse' first
[19:11:17] WARNING: Protection mode is ON. Unable to mount external drivers!
```

![image](https://user-images.githubusercontent.com/11080704/83363799-a51d9500-a372-11ea-9a8f-d3f52a1edbfd.png)

```yaml
workgroup: WORKGROUP
username: homeassistant
password: 'mypassword'
interface: ''
moredisks:
  - wd-drive
allow_hosts:
  - 10.0.0.0/8
  - 172.16.0.0/12
  - 192.168.0.0/16
  - 192.168.1.1/16
veto_files:
  - ._*
  - .DS_Store
  - Thumbs.db
  - icon?
  - .Trashes
```

If I use Portainer to open a console inside the add-on, I can see my drive:
```shell
bash-5.0# ls /dev_/disk/by-label/
EFI             hassos-boot     hassos-data     hassos-kernel   hassos-overlay  wd-drive
```
Trying to mount it there, throws the same error:
```shell
bash-5.0# mount -t auto /dev_/disk/by-label/wd-drive /wd-drive -o nosuid,relatime,noexec
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
fuse: device not found, try 'modprobe fuse' first
bash-5.0# modprobe fuse
modprobe: FATAL: Module fuse not found in directory /lib/modules/5.4.43
```

Any ideas on how to get it working?

**Comment by dianlight** [comment]
Exporting to exfat is quite problematic because the fuse driver is not stable. In these days I am taking back the project to make updates so stay connected but I can't promise anything.

Compatibility with Timemachine would not work on exFat in any case. (I don't know if you care)

If you can, I would advise you to switch to an ext3 / 4 file system on which everything works and is stable and fast.

L

**Comment by dianlight** [comment]
The new version work with exFat.
Can you try?

L.

**Comment by marciogranzotto** [comment]
@dianlight same issue on 9.2.0 ):

```
[22:07:23] WARNING: MoreDisk option found!
[22:07:23] INFO: More Disks mounting.. wd-drive
[22:07:23] INFO: Mount wd-drive
FUSE exfat 1.3.0
WARN: volume was not unmounted cleanly.
fuse: device not found, try 'modprobe fuse' first
[22:07:23] WARNING: Protection mode is ON. Unable to mount external drivers!
[cont-init.d] automount.sh: exited 0.
```

**Comment by marciogranzotto** [comment]
Well I formatted it on ext3 and is now working!

**Comment by dianlight** [comment]
I think is a problem with Hassios 4.8. I can’t try yet because the update don’t work on my installation.

In any case, the use of fuse with samba makes the share up to 5 times slower as it must write synchronously.  It is a known bug.

L.

**Comment by dianlight** [comment]
Fixed PR #10
